### DISCUSSION TITLE
这种情况怎么解决啊，显示我未登录，并且弹窗口

### DISCUSSION THREAD

**Discussion by cuijiaqi0406** [share121/inter-knot#5082] [委托]
![Screenshot_20240917_184220](https://github.com/user-attachments/assets/02b8d6e5-c614-4a2b-a66a-cc32b565b35b)

**Comment by Yizakl** [comment]
可以在设置切换用户接口喵

**Comment by cuijiaqi0406** [comment]
会了，谢谢！
